compiler/crates/relay-transforms/src/errors.rs

```diff
@@ -168,7 +168,7 @@ pub enum ValidationMessage {
     },
 
     #[error(
-        "No types implement the client interface {interface_name}. For a client interface to be used as a @RelayResolver @outputType, at least one Object type must implement the interface."
+        "No types implement the client interface {interface_name}. Interfaces returned by a @RelayResolver @outputType must have at least one concrete implementation."
```

Comment:
This validation now fires even for non-@outputType resolvers
```suggestion
        "No types implement the client interface {interface_name}. Interfaces returned by a @RelayResolver must have at least one concrete implementation."
```